omni_authify/providers/google.py

```diff
@@ -0,0 +1,61 @@
+import requests
+
+from urllib.parse import urlencode
+
+from .base import BaseOAuth2Provider
+
+
+class Google(BaseOAuth2Provider):
+    """
+    Google OAuth2 provider.
+    """
+    AUTHORIZE_URL: str = "https://accounts.google.com/o/oauth2/v2/auth"
+    PROFILE_URL: str = "https://www.googleapis.com/oauth2/v1/userinfo"
+
+    def __init__(self, client_id, client_secret, redirect_uri, scope="openid email profile"):
```

Comment:
Nega bo'tta scope ni static bribe qo'ydiz? ertaga library ishlatvotgan odam `openid email profile` she dfieldlardan boshqasini ololmidimi? 

defaults to `openid email profile` deb yozib qo'ysez bo'ladi, agar ulan scope ni yozishmasa, she fielder by default ishlasin deb. Commentni ham update qilish kk.